pipenv/environments.py

```diff
@@ -26,7 +26,7 @@
 PIPENV_DEFAULT_THREE = os.environ.get('PIPENV_DEFAULT_THREE')
 
 # Tells Pipenv to use the virtualenv-provided pip instead.
-PIPENV_USE_SYSTEM = os.environ.get('VIRTUAL_ENV') if not 'PIPENV_IGNORE_VIRTUALENVS' in os.environ else False
+PIPENV_USE_SYSTEM = os.environ.get('VIRTUAL_ENV') if 'PIPENV_IGNORE_VIRTUALENVS' not in os.environ else False
```

Comment:
this change is still apart of this commit batch, as I had to make changes to environments.py anyways.  Is that ok?